cncf_process/2022_karmada_annual/2022-karmada-annual.md

```diff
@@ -244,7 +244,7 @@ The Karmada project continues to grow and has its sight on short-term and long-t
 Multicloud means larger resource pools, which brings greater performance pressure to the multi-cloud management platform.
```

Comment:
* **Larger scalability**. We have been working on improving user experience in large-scale scenarios with the previous releases.
  Multicloud means larger resource pools, which brings greater performance pressure to the multi-cloud management platform.
  In large-scale scenarios, we will continue to reduce the access latency, resources consumed, and stability impact on the system under high concurrency.